client/extensions/woocommerce/state/wc-api/shipping-methods/actions.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Internal dependencies
+ */
+import request from '../request';
+import { setError } from '../../site/status/wc-api/actions';
```

Comment:
Kelly recently pointed out that we can avoid these relative URLs by doing

`woocommerce/state/site/status/wc-api/actions`, etc.